packages/itwinui-css/backstop/tests/alert.html

```diff
@@ -14,41 +14,22 @@
       content="width=device-width, initial-scale=1"
     />
     <title>Alert | iTwinUI</title>
-    <link
-      rel="stylesheet"
-      href="../../css/all.css"
-    />
-    <link
-      rel="stylesheet"
-      href="../assets/demo.css"
-    />
+    <script type="module">
+      import "./assets/theme.js";
```

Comment:
Why need it this way now? Something because of Vite?

Reply:
i did it this way to make sure vite knows the correct path